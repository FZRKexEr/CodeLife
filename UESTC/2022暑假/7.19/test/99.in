10
aeglhabfxiapdzqsawtkjeugawegyaxoirkopcchrccshkowoiowjnjrwtlxorbztrwgpyalzyfxhutmhgnjxonqxlqywxnidfja
4
mjaylzbeobjmotlcizdrsramdcxngsutcnfflyljppbodbjfwpscwgmzmpqibkyamyhhakouwupqtaexdqvmxtwwnswbkcwpqsqg
5
diemoidmeisrmkijlqeyizpfuxeihtdkdqmpokzzkcjednpoycpnrnsbmqhibyrxjapltocoewpxmesmedtanvpjazvnoiifcipk
1
mezqhkduthjqupuuqvtfvhmtfkwhtzutcywcphbhlozexpvbkdghhlkgdenibafqsqzmdbqnhyjasutrtfrawmxkiidjtxfuvsds
7
vrxypzzwcjrmroisrnhapldlruglxpfnjdjvlqeqlriwhdwonsafezunibdklljeesxzupinrsjibqmhlrnkkurnfbdbacekyngy
7
sgxzywjztqjryxwsnctpbdsprkrjatzifagvhbbcxfcqtqhishogfmrnssedkreigdefvjxmewrxwzzpyjtiokhmeplbhfmljpur
1
frkdtdyibecemthinhwvzdunkdcqhvtzwcxasctjealvowqrfmpfagppnxeeejahzvoecwuuoarxexdybadecheslnqucbmhtwao
10
bwesbuvyiucbsajvqsfslcpzokabkqguoayqwnctanzwplpcwzedqokjpujlostnerecpcftzvwzxnahdpkuksrrqtjgmskrlpdg
6
qlygubpopnkygniovbshslqahrghygsarkosdfqznnjnpoejkgoircmyduvcxtdtknlqqkcdvtercwmashjaoylxmbluhgupuong
6
aksrsuncbmjqjbotetvqxtpqpyssauofeyxpnptrgyrsozqavpduqqraooqukotkvqtyledqreldtfxsanqvafrxyydmkqhbfpax
5
