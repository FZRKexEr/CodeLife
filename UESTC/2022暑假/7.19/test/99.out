33554432
1048576
882499718
16384
16384
882499718
1024
65536
65536
1048576
